autogen/coding/local_commandline_code_executor.py

```diff
@@ -13,7 +13,8 @@
 
 
 __all__ = (
-    "LocalCommandlineCodeExecutor",
+    "LocalCommandLineCodeExecutor",
```

Comment:
One of it should be the old class name?

Reply:
I think it makes sense to just export the new names